website/blog/2021/11/26/apache-apisix-committer-experience.md

```diff
@@ -0,0 +1,48 @@
+
+So it all started from February, 2021. I was looking around GitHub to find OSS projects to contribute to, so as to use my knowledge for something that actually provides value to users. I would recommend people to check out the GSoC organisations page, try online searches or look for some GitHub repos that have a list of organisations for getting a starting point. Of Course you can contribute to projects randomly online, but I think this could be an efficient method. After looking around and testing multiple waters, I came across Apache APISIX. I looked up their JIRA and got in contact with [Zhiyuan Ju](https://github.com/juzhiyuan) (**PMC at Apache APISIX, and a great guy to work with**). After I got the slack invite from him to asf-slack, things started to get in motion. I picked up issues from the [apisix-website](https://github.com/apache/apisix-website) repository and worked along.
+
+## New perspective to APISIX
+
+While I was working on the website, I noticed that it was lacking on a lot of fronts and there was a wide gap between features that Apache APISIX provides and what was presented on the website. I discussed this with Zhiyuan and he agreed with my opinion. As [Google Summer of Code](https://summerofcode.withgoogle.com/archive/) was near, I proposed that I work on this project under this program. We both discussed this thoroughly and agreed on it. Through this project I met [Shuyang Wu](http://github.com/yiyiyimu), another awesome guy. Zhiyuan, Shuyang and Ming Wen were my mentors for this project. We started off in May and finally the new website was successfully up by September, 2021.
+
+![image](https://user-images.githubusercontent.com/57267960/143325043-acf485e1-2f07-4d3c-b693-debfd9fb788a.png)
+
+Being able to help the organisation by increasing its reach and improving its branding was a great experience for me. This has taken my experience at Apache APISIX to the next stage where I actively contribute to the organisation in areas of UX Engineering (UX Design + Frontend engineering), product and growth. My efforts being recognised and getting voted for a member and committer truly inspires me and I hope to do more on and on :)
+
+## Summarizing
+
+Lastly, for all the students or people starting out in OSS, my one piece of advice would be to try to love what you do and think more in terms of creating products. Why ? because to be able to create something that can be usable and useful for others, is I believe the main idea behind the existence of OSS. **Explore and find what you like and then obsess over it**.
+
+Thanks for reading along and your time ~
+> Shivam Singh<br/>
+> *<singhsh@apache.org>*<br/>
+> *<https://github.com/1502shivam-singh/>*<br/>
+> *<https://twitter.com/Shivam15_>*<br/>
```

Comment:
Maybe editing this into a sentence or two?

Reply:
You mean deleting the `>` right ?

Reply:
Not exactly...

I am a bit concerned about this part showing up in the page that do not look good. You can try `cd website`, then `yarn start` to run the blog locally. If these lines look good to you, just leave them as they are.

Another option is to edit them into one or two sentences like:

"Here is my email, Github, and Twitter, feel free to contact me and we can discuss some interesting topics."

Reply:
Okay, I got what you meant. Using hyperlinks now, instead of full links displayed straight up.

Reply:
Ok cool. Other parts LGTM.